### PR TITLE
Countries => regions visited / lived in

### LINKED ISSUE #1464: Frontend uses "countries" lived/visited instead of "regions"
Title. So that Couchers doesn't arbitrate what is or isn't a country

### PR DESCRIPTION
**Pull request by rbrishabh**
fixes #1464

hey @ShmuelTreiger, how does this look? Let me know if this needs any changes! Thanks

### CODE REVIEW

**Comment by lucaslcode** [comment]
Thanks! Did you check all the tests are passing with `yarn test`?

File: app/frontend/src/features/constants.ts [hunk 1]
```diff
@@ -47,8 +47,8 @@ export const SECTION_LABELS_A11Y_TEXT = "tabs for user's details";
 
 // Edit Profile
 export const ACCOUNT_SETTINGS = "Account Settings";
-export const COUNTRIES_VISITED = "Countries I've Visited";
-export const COUNTRIES_LIVED = "Countries I've Lived In";
+export const COUNTRIES_VISITED = "Regions I've Visited";
```
Comment (lucaslcode) [inline app/frontend/src/features/constants.ts]: It would be better to change the constant name too.

**Comment by ShmuelTreiger** [comment]
In about.tsx, "country" is used in a bunch of variable names. Do those need changing? (I don't know much about the front end)

**Comment by rbrishabh** [comment]
hey @lucaslcode! How does this look now?
Thanks!

**Review by lucaslcode — CHANGES_REQUESTED** [review]
Please run `yarn lint --fix` to sort imports

**Comment by rbrishabh** [comment]
Sure!

**Comment by rbrishabh** [comment]
done! @lucaslcode Feel free to review when you have time!

**Review by lucaslcode — APPROVED** [review]
Thanks for your contribution!

**Pull request merged**